executor/ddl.go

```diff
@@ -194,6 +197,14 @@ func (e *DDLExec) executeDropTable(s *ast.DropTableStmt) error {
 			return errors.Trace(err)
 		}
 
+		if os.Getenv("TIDB_CHECK_BEFORE_DROP") == "1" {
```

Comment:
Please add a log here. I suggest using the warn log level.

Reply:
done.